src/main/java/laledevri/laladevri/App.java

```diff
@@ -8,6 +8,20 @@
 {
     public static void main( String[] args )
     {
+
+int target2 = -5;
+
+
+
+	int num3 = 3;
+
+
+
+final int deneme2;
+
+
+
+	target2 =- num3;  // Noncompliant; target = -3. Is that really what's meant?
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Was "-=" meant instead? [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS2757)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "target2". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS1854)